modules/broker-amqp/src/main/java/org/wso2/broker/amqp/AmqpConsumer.java

```diff
@@ -57,25 +67,31 @@ public AmqpConsumer(ChannelHandlerContext ctx, AmqpChannel channel,
         this.context = ctx;
         this.channel = channel;
         this.errorLogger = new ErrorLogger(queueName);
+        this.tracingProperties = new ArrayList<>();
```

Comment:
Shall we initialize this to 2 if we only have two entries now?

Reply:
Added.